_config.yml

```diff
@@ -3,12 +3,17 @@ email:             your-email@example.com
 description:       A Blog Theme by Start Bootstrap
 author:            Start Bootstrap
 baseurl:           "/startbootstrap-clean-blog-jekyll"
-url:               "http://blackrockdigital.github.io"
+url:               "http://amanabt.github.io"
```

Comment:
I don't think you should change the sites URL here

Reply:
I changed the site URL temporarily so as to host a demo website. If you are fine with all other changes, then I will revert the `url` to `http://blackrockdigital.github.io` and put up a `READY TO MERGE` flag.

Reply:
ah ok - sorry. I'm not a member of this repo, but just recently discovered this theme and was looking through the PRs. This looks like a good one so hopefully @davidtmiller @BlackrockDigital can review and merge. :-)